react/features/chrome-extension-banner/components/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as ChromeExtensionBanner }
+    from './ChromeExtensionBanner';
```

Comment:
If this is not supposed to be used or shown on mobile. Make it web-specific.
Rename ChromeExtensionBanner.js to ChromeExtensionBanner.web.js and create an empty file ChromeExtensionBanner.native.js.

Reply:
done